### DISCUSSION TITLE
Feature request: Autocomplete of links

### DISCUSSION THREAD

**Discussion by AndrewMWebster** [nothingislost/obsidian-hover-editor#221] [Ideas]
As a Hover Editor user
I want to be able to click or press ENTER to autocomplete Obsidian links
So that the editing behavior within the Hover Editor is consistent with the behavior in the main editor windows and I can maintain flow while working.

This keeps snagging me, and I'd love it if I can start a link, the search function find it, and I can just click it or down-arrow and ENTER to put the link in.  Hope this is possible!

**Comment by pjeby** [comment]
I don't understand this request, since Hover Editor doesn't in any way interfere with this normal Obsidian behavior.  If it's not working for you, the first thing to check would be whether you can reproduce the problem in a fresh vault with the stock theme and no other plugins or snippets -- and if so, then file a bug report in the issues section where it can be tracked.

**Comment by AndrewMWebster** [comment]
I created a clean, fresh vault, added in only the Hover Editor, set it to
default to the Edit view, and it was fine.

It'll take me a little while (a day to two) but I'll try now to recreate
the vault set-up where it's not working, and let you know if I can figure
out the clash.  You never know, it might be something fixable if we know
what to fix!
